ui/page/dexclient/create_wallet_modal.go

```diff
@@ -90,10 +90,12 @@ func (md *createWalletModal) Dismiss() {
 
 func (md *createWalletModal) OnDismiss() {
 	md.ctxCancel()
+	md.sourceAccountSelector.UnsubscribeTxNotifications()
```

Comment:
Explains a previous comment about automatically unregistering the listener when the ctx is canceled. Don't need to be a separate method.